src/index.html

```diff
@@ -16,17 +17,29 @@
       href="style.css"
```

Comment:
The attributes of the `link` tag should be started on new lines with a 2-space indentation relative to the tag to keep the attributes correctly formatted.